sample/HttpWorker/host.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Can you make the terminology consistent here? HttpWorker/HttpInvoker. Perhaps from the public user point of view the feature should be called "HttpWorker" rather than invoker, since invoker is our own internal concept, and the concept of "Worker" (e.g. LanguageWorker) is already surfaced to users to a degree?

So the config section would be httpWorker, and the configuration points to/configures the worker implementation (exe). That makes sense. It really isn't an invoker.

Reply:
done! Switched to HttpWorker every where